Sources/Recycling/ListViews/AdManagement/Cells/UserAdManagementStatisticsCell.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Copyright © FINN.no AS. All rights reserved.
+//
+
+public class UserAdManagementStatisticsCell: UITableViewCell {
+    public var itemModels = [StatisticsItemModel]() {
+        didSet { updateStackViewContent() }
+    }
+    var stackView: UIStackView = {
+        let view = UIStackView(frame: .zero)
+        view.alignment = .fill
+        view.distribution = .fillEqually
+        return view
+    }()
+    var separatorView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .sardine
+        return view
+    }()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    private func updateStackViewContent() {
+        for oldSubview in stackView.arrangedSubviews {
+            stackView.removeArrangedSubview(oldSubview)
+        }
+        let lastIndex = itemModels.count-1
+
+        for (index, model) in itemModels.enumerated() {
+            let itemView = StatisticsItemView()
+            itemView.itemModel = model
+            itemView.shouldShowLeftSeparator = index > 0 && index < lastIndex
+            itemView.shouldShowRightSeparator = index > 0 && index < lastIndex
+            // Slight overengineering, as there are currently no plans to receive any other number than three items
+            if lastIndex == 1 { itemView.shouldShowLeftSeparator = index == lastIndex }
+            if lastIndex > 2 { itemView.shouldShowRightSeparator = index == lastIndex-1 }
+            stackView.addArrangedSubview(itemView)
+            itemView.setup()
+        }
+    }
+
+    func setup() {
+        contentView.addSubview(separatorView)
+        separatorView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Can we place this in the property declaration instead?

Reply:
You may also `UIView(withAutoLayout: true)` instead. Both for `separatorView` and `stackView`.